src/constants.ts

```diff
@@ -0,0 +1,8 @@
+export type IRouteKey = 'home' | 'lobby' | 'room' | 'game';
+
+export const ROUTES: Record<IRouteKey, string> = {
```

Comment:
Little Typescript improvement:

Instead of doing `interface IRecord { [key: string]: string }` instead you can use the built-in type `Record<string, string>`. This saves the hassle of dealing with index types as well as it's easier to write.